cmake/partition_manager.cmake

```diff
@@ -522,4 +522,22 @@ else()
   set(ZEPHYR_RUNNER_CONFIG_KERNEL_HEX "${final_merged}"
     CACHE STRING "Path to merged image in Intel Hex format" FORCE)
 
+  # Zephyr has a Kconfig option used for signing an application image
```

Comment:
Wouldn't `mcuboot.cmake` be a better place for this logic?

Reply:
@hakonfam you apparently didn't see: https://github.com/nrfconnect/sdk-nrf/pull/3092#issuecomment-706370317

Reply:
I assumed your comments were in the PR, so no I didn't see that, good to see that we agree :)